### PR TITLE
Fix Graphics _localBounds calculation

### PR DESCRIPTION
**Pull request by kib357**
Graphics bounds calc was broken after adding `lineAlignment` option for non-default values of `lineAlignment`. For example, this is what we getting when value is `1` (outer lines):
image
And this is for `0` (inner lines):\
image
Code in PR resolves this issue.

### CODE REVIEW

**Comment by ivanpopelyshev** [comment]
I'm gonna make fix for polygon, because there's no guarantee about its orientation.

File: src/core/graphics/Graphics.js [hunk 1]
```diff
@@ -1112,15 +1112,27 @@ export default class Graphics extends Container
...
+
+                let lineOffset = 0;
+
+                if (lineAlignment === 0.5)
+                {
+                    lineOffset = lineWidth / 2;
+                }
+                else if (lineAlignment === 1)
+                {
+                    lineOffset = lineWidth;
```
Comment (bigtimebuddy) [inline src/core/graphics/Graphics.js]: lineAlignment should be treated as a float from 0-1. So probably lineOffset = lineWidth * lineAlignment instead.

File: src/core/graphics/Graphics.js [hunk 2]
```diff
@@ -1112,15 +1112,27 @@ export default class Graphics extends Container
                 const data = this.graphicsData[i];
                 const type = data.type;
                 const lineWidth = data.lineWidth;
+                const lineAlignment = data.lineAlignment;
+
+                let lineOffset = 0;
```
Comment (bigtimebuddy) [inline src/core/graphics/Graphics.js]:
```suggestion
                const lineOffset = lineWidth * lineAlignment;
```

Comment (kib357) [reply src/core/graphics/Graphics.js]: Thank you, good point!

**Comment by kib357** [comment]
> I'm gonna make fix for polygon, because there's no guarantee about its orientation. Alignment 0 and 1 is the same there.

@ivanpopelyshev Bounds calculation is correct for non-clockwise oriented polygons, but rendering is consider this polygons as turned inside out. So i don't now what should be the right behaviour...

**Comment by ivanpopelyshev** [comment]
> @ivanpopelyshev Bounds calculation is correct for non-clockwise oriented polygons, but rendering is consider this polygons as turned inside out. So i don't now what should be the right behaviour...

It's correct for clockwise, its incorrect for non-clockwise, i will prove it tomorrow :)

**Comment by bigtimebuddy** [comment]
Thank you for this change @kib357! @ivanpopelyshev, if you want to do a follow-up PR to consider the direction of the polygon, that's welcomed, but I think this is a good improvement over the existing local bounds.

**Pull request merged**